plugins/rhacm/src/components/ClusterStatusPage/ClusterStatusPage.tsx

```diff
@@ -83,8 +84,13 @@ const CatalogClusters = () => {
       const clusterResourceEntities = await catalogApi.getEntities({
         filter: { kind: 'Resource', 'spec.type': 'kubernetes-cluster' },
       });
+
       const clusters = await getClusters(configApi);
 
+      if ('error' in clusters) {
+        throw new Error((clusters as unknown as ErrorResponseBody).error.message)
```

Comment:
When you have to do this type of casting, it tells you something is wrong somewhere. Why can't you cast `clusters` directly to `ErrorResponseBody`? Think about it :slightly_smiling_face:

Reply:
Yea I though there had to be a better way. Updated in the latest commit.